cmd/podman/runlabel.go

```diff
@@ -179,3 +180,66 @@ func runlabelCmd(c *cliconfig.RunlabelValues) error {
 
 	return utils.ExecCmdWithStdStreams(stdIn, stdOut, stdErr, env, cmd[0], cmd[1:]...)
 }
+
+func getGlobalOpts(c *cliconfig.RunlabelValues) string {
```

Comment:
@mheon, should this live in a common or util file somewhere?

Reply:
I've moved it to package util